crates/libs/windows/src/core/mod.rs

```diff
@@ -95,3 +95,32 @@ fn wide_trim_end(mut wide: &[u16]) -> &[u16] {
     }
     wide
 }
+
+#[doc(hidden)]
```

Comment:
Despite this being not shown to end users, it could be good to have some documentation here for those working on the internals of windows-rs.